app/src/main/res/layout/logins_item.xml

```diff
@@ -15,34 +12,39 @@
     android:focusable="true"
     android:minHeight="?android:attr/listPreferredItemHeight">
 
-    <ImageView
-        android:id="@+id/favicon_image"
-        android:layout_width="24dp"
-        android:layout_height="24dp"
+    <FrameLayout
```

Comment:
I think `android:importantForAccessibility="noHideDescendants"` should be added to this to ensure it and children will not be read by accessibility services (same as what is happening now).